### PR TITLE
Don't apply Kover to projects with AAR libraries

### PR DESCRIPTION
**Pull request by jmartinesp**
## Type of change

- [ ] Feature
- [ ] Bugfix
- [x] Technical
- [ ] Other :

## Content

Do not apply the Kover plugin to modules with only AAR libraries.

## Motivation and context

Without this we can't test local versions of the Rust SDK or the RTE library.

## Tests

- Try building the app using a local version of either the RTE or the Rust SDK.

## Tested devices

- [ ] Physical
- [ ] Emulator
- OS version(s):

## Checklist

- [ ] Changes have been tested on an Android device or Android emulator with API 23
- [ ] UI change has been tested on both light and dark themes
- [ ] Accessibility has been taken into account. See https://github.com/element-hq/element-x-android/blob/develop/CONTRIBUTING.md#accessibility
- [x] Pull request is based on the develop branch
- [ ] Pull request includes a new file under ./changelog.d. See https://github.com/element-hq/element-x-android/blob/develop/CONTRIBUTING.md#changelog
- [ ] Pull request includes screenshots or videos if containing UI changes
- [ ] Pull request includes a [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off)
- [x] You've made a self review of your PR

### CODE REVIEW

File: app/build.gradle.kts [hunk 1]
```diff
@@ -28,7 +28,7 @@ plugins {
     alias(libs.plugins.anvil)
     alias(libs.plugins.ksp)
     alias(libs.plugins.kapt)
-    alias(libs.plugins.firebaseAppDistribution)
+    id(libs.plugins.firebaseAppDistribution.get().pluginId)
```
Comment (ganfra) [inline app/build.gradle.kts:31]: Why this change?

**Review by ganfra — APPROVED** [review]

File: app/build.gradle.kts (hunk 1 above)
Comment (jmartinesp) [reply app/build.gradle.kts:31]: Otherwise there is an issue with the firebase BOM, it keeps throwing an error about the dependency already existing in the classpath with another version or something similar.
